scripts/deployments/start_without_deployment_agent.bash

```diff
@@ -92,7 +92,7 @@ else
     export DOCKER_IMAGE_TAG=development;
     export DEV_PC_CPU_COUNT=8;
     # Download yq utility
-    python -c "import urllib.request,os,sys,urllib; f = open(os.path.basename(sys.argv[1]), 'wb'); f.write(urllib.request.urlopen(sys.argv[1]).read()); f.close();" https://github.com/mikefarah/yq/releases/download/v4.29.2/yq_linux_amd64
+    python3 -c "import urllib.request,os,sys,urllib; f = open(os.path.basename(sys.argv[1]), 'wb'); f.write(urllib.request.urlopen(sys.argv[1]).read()); f.close();" https://github.com/mikefarah/yq/releases/download/v4.29.2/yq_linux_amd64
```

Comment:
Are you sure about this?
This might not be running in the correct environment.
Since I am not familiar with this pice of code I cannot tell you what should be correct.

Reply:
question here: is this supposed to run in a virtual environment?
cause then it should use something like ```.venv/bin/python```
actually if you are in a virtual env of python3, ```python``` is already python3

Reply:
So yes, I tend to agree with SAN here, you might want to check where you run these. I think that `python` is more portable than `python3`.
### **But this applies if your run in a virtualenviornment.**

Reply:
I reverted the changes. Now, we use `python` in these 2 scripts